<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Disseminate</class>
 <widget class="QMainWindow" name="Disseminate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Disseminate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="configurationGroup">
      <property name="title">
       <string>Configuration</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="configuration"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="clientGroup">
        <property name="title">
         <string>Clients</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="clientList"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushSettings">
           <property name="text">
            <string>Push Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reloadClients">
           <property name="text">
            <string>Reload Clients</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="keysGroup">
        <property name="title">
         <string>Keys</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="keyList"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="addKey">
             <property name="text">
              <string>Add Key</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeKey">
             <property name="text">
              <string>Remove Key</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="whitelistRadio">
             <property name="text">
              <string>Whitelist</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="blacklistRadio">
             <property name="text">
              <string>Blacklist</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuHello"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionTemplates"/>
   <addaction name="separator"/>
   <addaction name="actionAddConfiguration"/>
   <addaction name="actionRemoveConfiguration"/>
   <addaction name="actionEditConfiguration"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/ic_fiber_manual_record_black_48dp.png</normaloff>:/icons/icons/ic_fiber_manual_record_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start Broadcasting</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/ic_stop_black_48dp.png</normaloff>:/icons/icons/ic_stop_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Broadcasting</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionTemplates">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/ic_content_copy_black_48dp.png</normaloff>:/icons/icons/ic_content_copy_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>Templates</string>
   </property>
  </action>
  <action name="actionAddConfiguration">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/ic_add_black_48dp.png</normaloff>:/icons/icons/ic_add_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>Add Configuration</string>
   </property>
  </action>
  <action name="actionRemoveConfiguration">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/ic_remove_black_48dp.png</normaloff>:/icons/icons/ic_remove_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>Remove Configuration</string>
   </property>
  </action>
  <action name="actionEditConfiguration">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/ic_edit_black_48dp.png</normaloff>:/icons/icons/ic_edit_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>Edit Configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
